components/AppBar/examples/AppBarModalPresentationExample.swift

```diff
@@ -28,8 +28,6 @@ class AppBarModalPresentationSwiftExamplePresented: UITableViewController {
 
     self.addChildViewController(appBar.headerViewController)
 
-    let color = UIColor(white: 0.2, alpha:1)
-    appBar.headerViewController.headerView.backgroundColor = color
     let mutator = MDCAppBarTextColorAccessibilityMutator()
     mutator.mutate(appBar)
```

Comment:
If you move the `mutate` call to `-viewDidAppear:`, then any UIAppearance values will be set on the instance. 

If our examples are to demonstrate how to work with AppBars and UIAppearance together, then we should move all of the mutators into `-viewDidAppear`.

Reply:
Interesting point.  For some reason I had it in my head that viewDidAppear was too late in certain cases, but I'm not sure my reasoning.  Andrés?

Reply:
'viewDidAppear' is indeed too late, the wrong colors are visible while the view animates in.